CONTRIBUTING.md

```diff
@@ -1,6 +1,8 @@
-## Summary of upstream dockerfiles
+# kserve's server rocks
```

Comment:
Tiny nit: upstream writes it as `KServe` in their [docs](https://kserve.github.io/website/0.11/), maybe we should follow that.

Reply:
Fixed!